docs/insomnia/http-proxy.md

```diff
@@ -5,10 +5,19 @@ category: "Get Started"
 category-url: get-started
 ---
 
-Insomnia does not automatically detect system-wide proxy settings but a proxy can be set up manually if needed. From the settings menu, set the HTTP Proxy and/or HTTPS Proxy to the location of your proxy server and all future requests will be rerouted.
+Insomnia does not automatically detect system-wide proxy settings. A proxy can be set up manually. Set your HTTP or HTTP proxy server and reroute all future requests through that server by accessing your Settings menu > **General** > **HTTP Network Proxy**.
```

Comment:
The second "HTTP" needs to be "HTTPS". 

Should we use "Settings menu" or "Preferences"? The app says Preferences, not sure what we're using in docs for consistency.

Reply:
Let's opt for Preferences, as that's what the app says.